src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -1229,6 +1229,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageService
             Return node.IsKind(SyntaxKind.InferredFieldInitializer)
         End Function
 
+        Public Function IsInferredTupleMemberDeclarator(node As SyntaxNode) As Boolean Implements ISyntaxFacts.IsInferredTupleMemberDeclarator
+            ' TODO: Evaluate if we need any further processing
+            Return node.IsKind(SyntaxKind.TypedTupleElement)
+        End Function
+
+        Public Function GetAssignedExpressionForAnonymousTypeDeclarator(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetAssignedExpressionForAnonymousTypeDeclarator
+            ' TODO: Implement
+            Return Nothing
+        End Function
+
+        Public Function GetAssignedExpressionForTupleMemberDeclarator(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetAssignedExpressionForTupleMemberDeclarator
+            ' TODO: Implement
+            Return Nothing
+        End Function
+
```

Comment:
The TODOs are left to indicate that this PR does not implement the finder for VB. Do we leave these as they are? Do we need to include tests showing the behavior for VB?